README.md

```diff
@@ -1,2 +1,11 @@
 # password-generator
-Built a password generator focusing on Javascript
+
+Built a password generator focusing on Javascript.
+
```

Comment:
Please add in a description here about the project and what it does. You could talk about the code you've written, how the functions work etc. Please add this in.

Reply:
have updated with brief summary of function